pkg/cloudprovider/provider/openstack/provider.go

```diff
@@ -733,3 +754,42 @@ type forbiddenResponse struct {
 		Code    int    `json:"code"`
 	} `json:"forbidden"`
 }
+
+func (p *provider) cleanupFloatingIP(machine *v1alpha1.Machine, updater cloud.MachineUpdater) error {
+	var floatingIPID string
+	if val, exists := machine.Annotations[floatingIPIDAnnotationKey]; exists {
+		floatingIPID = val
+	} else {
+		return osErrorToTerminalError(fmt.Errorf("failed to release floating ip"),
+			fmt.Sprintf("%s finalizer exists but %s annotation does not", floatingIPReleaseFinalizer, floatingIPIDAnnotationKey))
+	}
```

Comment:
This can be simplified with:
```go
floatingIPID, exists := machine.Annotations[floatingIPIDAnnotationKey]
if !exists {
    return osErrorToTerminalError(fmt.Errorf("failed to release floating ip"),
			fmt.Sprintf("%s finalizer exists but %s annotation does not", floatingIPReleaseFinalizer, floatingIPIDAnnotationKey))
}
```

Reply:
Done